src/ui/reducers/timeline.ts

```diff
@@ -140,3 +149,8 @@ export const isMaximumFocusWindow = (state: UIState) => {
     return false;
   }
 };
+
+export const getRecordingMightHaveRoutines = (state: UIState) => {
```

Comment:
Naming nit: I think it might make more sense if this was called `getRecordingTooLongToSupportRoutines` then at least the relationship I mentioned above would seem more clear.